### DISCUSSION TITLE
Organisation profile for Enterprise Orgs

### DISCUSSION THREAD

**Discussion by Timmmy-nlb** [community/community#15306] [Enterprise]
The "Organisation Profile" function introduced yesterday is great.
So you can create a better overview for the organization.

BUT: 
Why are the enterprise organizations forgotten again?
It would have been a great thing if you could create a `.github-internal` repo, with the same function for all organization members of the Enterprise organization.

**Comment by bisdevres** [comment]
need this too.
[this is a similar feature request](https://github.com/community/community/discussions/8461)
